index.html

```diff
@@ -386,6 +386,30 @@
           a representation of the signing date.</dd>
         </dl>
       </section>
+
+      <section>
+        <h2>Revocation Model</h2>
+        <p>A <a>Verifiable Claim</a> may provide revocation information
+        by referring to a revocation list by adding the following
+        <a>property</a> to the <a>Verifiable Claims Model</a>:</p>
+        <dl>
+          <dt><var>revocationList</var></dt>
+          <dd>The format of the revocation list will vary depending on the
+          signature scheme used, but will almost always be a URL to a
+          resource that contains zero or more revocation entries.
+          </dd>
+        </dl>
+
+        <p class="issue" data-number="35" data-title="Define ONE concrete format for the revocationList parameter ">
+What is the format of the revocation list? Options include 1)
```

Comment:
"What is the format of the revocation list" => "What is the format of a simple revocation list?"